src/Tests.IntegrationTests/Fixtures/CentOpsFixture.cs

```diff
@@ -66,7 +66,7 @@ public CentOpsFixture(IConfiguration configuration)
                 Host = "http://bot1/dmr-api/messages",
                 Type = "Chatbot",
                 Status = "Active",
-                ApiKey = "thisisareallylongkey"
+                ApiKey = "thisisareallylongkeyformockbot1"
```

Comment:
I can see in the docker compose file that you've added keys for both mockbots, do you also need a key for mockbot2 here in the code?

Reply:
The test doesn't use bot2 currently - but I figured I'd ensure the API key was unique as I was in there.

Reply:
I keep forgetting that the scenario involving mockbot2 has not yet been lit up in the tests.  The person writing that scenario can add in the code for mockbot2 here.

Reply:
We talked about this - bot2 will come into play in future tests, but for now it's there as a placeholder.